src/test/resources/fixtures/hmh/hmh-interacted/development/caliperEventAssessementStarted.json

```diff
@@ -0,0 +1,185 @@
+{
+    "sensor": "http://rce.localhost/sensor/1",
+    "sendTime": "2019-06-10T14:50:47.341Z",
+    "dataVersion": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "data": [
+        {
+            "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+            "id": "urn:uuid:13ca9c97-80f8-4136-ac84-532799feda47",
+            "type": "AssessmentEvent",
+            "actor": {
+                "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                "id": "urn:uuid:ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+                "type": "Person",
+                "name": null,
```

Comment:
no need to provide these fields if they are null. if a field is null, just leave it out (same applies for the ones below) thanks.

Reply:
Dear @odowdj the fields mentioned above belongs to RCE, the interactionEd only sends the Object and the RCE wraps it and send the event. And I believe @neil-buckley had fixed it!